pages/osx/xar.md

```diff
@@ -0,0 +1,15 @@
+# xar
+
+> Manage .xar archives.
+
+- Create a xar archive of all files in a given directory:
+
+`xar -cf {{archive.xar}} {{path/to/directory}}`
+
+- List the contents of a given xar archive:
+
+`xar -tf {{archive.xar}}`
```

Comment:
Let's use `--list` instead of `-t`

Reply:
I tried but `--list` doesn't work. There's also no such usage [here](https://linux.die.net/man/1/xar).

Reply:
This page https://mackyle.github.io/xar/xar.html mentions that `--list` is a synonym for `-t`. Is it a version related issue ? Or the doc is incorrect ?

Reply:
It looks like it was originally in subversion (the old version) and then forked to GitHub and updated.

Reply:
Even the github page has the `--list` option - https://github.com/mackyle/xar/blob/master/xar/src/xar.1#L29

Reply:
Yeah, the new versions do. The man page on Linux.die is the original older version.

Reply:
```
$ which xar
/usr/bin/xar
```
Does this mean it's preinstalled on mac?

Reply:
Not necessarily. Any package is free to install a command at `/usr/bin`. Is there a brew version of xar that you can get the latest version of ?

Reply:
```
$ brew search xar
==> Formulae
xar-mackyle                              xaric
$ brew info xar-mackyle
xar-mackyle: stable 1.6.1 (bottled)
eXtensible ARchiver
https://mackyle.github.io/xar/
Not installed
```
I just checked my friend's mac. It also has `xar`.  I'm guessing `xar` comes preinstalled on mac and the github version of `xar` is different from the one on mac.
Also, mac version:
```
$ xar --version
xar 1.8dev
```

Reply:
Ok, so the mac version is more recent than the brew version ? And that still doesn't have the --list option ?

Reply:
It seems so. Should I remove the GitHub link as homepage?

Reply:
Sure. Then put this https://linux.die.net/man/1/xar as the home page. And change `list` to `lis[t]`

Reply:
Sure, changed it